### DISCUSSION TITLE
Feature: Add ability to create Docker networks

### DISCUSSION THREAD

**Discussion by vitor-mariano** [IceWhaleTech/CasaOS#411] [Ideas]
**Is your feature request related to a problem? Please describe.**
Currently, all the containers are created and connected to the default Docker network. This prevents one container to connect to another one. The problem is that in some cases it's necessary to connect some of them, for example, to a Postgres container.

**Describe the solution you'd like**
Add a network option to the app.

**Describe alternatives you've considered**
The workaround is to create a network and connect my containers manually through the Terminal.

```sh
docker network create casaos
docker network connect casaos <container_name>
```

**Comment by Brottus** [comment]
Hejo @Casaos Team,

having the same issue like @vitor-mariano . I could imagine that we could have a small button next to the settings button for such networks in which we can create these by simply typing names in it. Once that list is created if you choose for example bridged network in the Container app menu another drop down menu would pop up to either add one of your networks to it or leave it blank so the container would not be added. Or just add it to the dropdown menu which is already there for bridged and host.
Hope to see this feature come to live.
